filter.imp

```diff
@@ -96,21 +96,9 @@
 
 
    # The below headers were added by hand due to errors being found when the private headers were included.
```

Comment:
How do I know when to add an asm -> linux mapping to this file vs the newly created nonarch.imp? Perhaps additional comments in either file will make it clearer?